readme.md

```diff
@@ -104,8 +108,8 @@ Your Answer:
 
 Write some basic CSS that demonstrates changing a CSS property when the device width drops below 40rem.
 
-```css
-
+```
+	N/A
```

Comment:
Would love to see you make an attempt at this, even you don't know the syntax, pseudo code the steps.  Would recommend you review the [lesson plan](https://github.com/ga-dc/curriculum/tree/master/08-single-page-web-apps/responsive-web-design), and then come back to this.